scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/common/util/logforwarder/util/LoggingOutputStream.java

```diff
@@ -160,6 +160,40 @@ public void write(final int b) throws IOException {
         count++;
     }
 
+    private void flush(boolean waitForNewLine) {
+        if (this.count != 0) {
+            if (this.count == lineSepBytes.length) {
```

Comment:
this code could be refactored in a private function boolean checkBufferIsLineSeparator()